app/services/container_dashboard_service.rb

```diff
@@ -123,6 +123,7 @@ def heatmaps
     # Get latest hourly rollup for each node.
     node_ids = @ems.container_nodes if @ems.present?
     metrics = MetricRollup.latest_rollups(ContainerNode.name, node_ids)
+    metrics = metrics.where('timestamp > ?', 1.day.ago.utc).includes(:resource)
```

Comment:
not trying to go too deep on this one

1. include and distinct are either both needed or neither are needed.
2. it looks like includes is probably not needed in latest_rollups (so neither are needed there)

pulling them out into their own scope seems to make sense. Agreed that only bringing back metric resource name would be optimal, and agreed that you should not do that and punt for now.